src/app/sentry/SentryProvider.tsx

```diff
@@ -0,0 +1,63 @@
+import { Profile } from "@arkecosystem/platform-sdk-profiles";
+import * as Sentry from "@sentry/react";
+import { Integrations } from "@sentry/tracing";
+import React, { useCallback, useContext, useRef } from "react";
+
+import { SentryErrorBoundary } from "./SentryErrorBoundary";
+
+const SentryContext = React.createContext<any>(undefined);
+
+const useSentry = () => {
+	const initializedProfileRef = useRef<Profile>();
+
+	const initSentry = useCallback((profile: Profile) => {
+		if (initializedProfileRef.current?.id() === profile.id()) {
+			return;
+		}
+
+		initializedProfileRef.current = profile;
+
+		Sentry.init({
+			dsn: process.env.REACT_APP_SENTRY_DSN,
+			integrations: [new Integrations.BrowserTracing()],
+			tracesSampleRate: 1.0,
+		});
+
+		Sentry.setContext("profile", {
```

Comment:
A few things we could capture here to provide more details, all anonymous information.

- Profile ID
- Profile Settings (excluding things like passwords)
- Profile Networks
- Number of Wallets
- Was ledger used when it errored
- Last used wallet ID (if available)